woocommerce/woocommerce
# Update Pattern Thumbnail Shadow

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #41158 .

Updates the shadow treatment on pattern thumbnails in Customize Your Store to make them clearer.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new WooCommerce installation with this version.
1. Make sure to enable customize-store feature flag
1. Go to /wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store%2Fassembler-hub
1. Go to "Change your header"
1. See that the pattern thumbnail shadow is the same as the screenshot in issue #41158
1.  Go to "Change your Change your homepage"
1. See that the pattern thumbnail shadow is the same as the screenshot in issue #41158
1.  Go to "Change your Change your footer"
1. See that the pattern thumbnail shadow is the same as the screenshot in issue #41158




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
